api/interventions/models.py

```diff
@@ -8,7 +8,7 @@ class InterventionsModel(models.Model):
     The interventions model
     """
 
-    createdBy = models.IntegerField()
```

Comment:
This shoud be `    createdBy = models.IntegerField()
` and not the one you have accepted. CreatedBy should be an integer

Reply:
Remember 'createdBy' should return a string of 'username' when retrieving. Setting it to an integer will not allow for this.

Reply:
When POSTing, it should return an integer, then when retrieving fetch the username based on the id field

Reply:
It still returns the id of the user when posting an intervention and the username when retrieving the record. If we change the createdBy serializer field to an integer, during migrations the DB registers this as an integer hence no way of returning a string when retrieving.